helix: add example to open multiple files and tutorial (#14675)
tldr-pages/tldr: pages/common/helix.md

`helix {{path/to/file}}`

- Open files and show them one next each other:

`helix --vsplit {{path/to/file1 path/to/file2}}`

> - Show the tutorial to learn to use Helix. Alternatively inside helix (press <Esc>:tutor):
> 
> `helix --tutor`

Reviewer: ```suggestion
- Show the tutorial to learn Helix (or access it within Helix by pressing `<Esc>` and typing `:tutor`):

`helix --tutor`
```
